File: 1574-MaximumProductOfTwoElementsInAnArray/1574-MaximumProductOfTwoElementsInAnArray.java
// Last updated: 5/26/2026, 11:18:13 AM
class Solution {
    public int maxProduct(int[] nums) {
        
        Arrays.sort(nums);
        int n = nums.length-1;

        int result = (nums[n]-1) * (nums[n-1]-1);
        return result;
      
    }
}

File: 1574-MaximumProductOfTwoElementsInAnArray/1574-MaximumProductOfTwoElementsInAnArray.py
# Last updated: 5/26/2026, 11:18:08 AM
class Solution(object):
    def maxProduct(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """

        nums.sort()
        return (nums[-1]-1) * (nums[-2]-1)
        